index.html

```diff
@@ -99,77 +138,90 @@ <h3 class="projects-title">Project names goes here</h3>
-          <li class="template-item">Ruby on rails</li>
-          <li class="template-item">RSpec</li>
-          <li class="template-item">Capibara</li>
-        </ul>
-      </div>
+        <div id="frameworks-template">
+          <p class="templates-title">Frameworks</p>
+          <ul class="list-container">
+            <li class="template-item">React.js</li>
+            <li class="template-item">Ruby on rails</li>
+            <li class="template-item">RSpec</li>
+            <li class="template-item">Capibara</li>
+          </ul>
+        </div>
         
-      <div id="skills-template">
-        <p class="templates-title">Skills</p>
-        <ul class="list-container">
-          <li class="template-item">Database Management</li>
-          <li class="template-item">CLI</li>
-          <li class="template-item">API Design</li>
-          <li class="template-item">Version Control</li>
-          <li class="template-item">Web Development</li>
-        </ul>
-      </div>
+        <div id="skills-template">
+          <p class="templates-title">Skills</p>
+          <ul class="list-container">
+            <li class="template-item">Database Management</li>
+            <li class="template-item">CLI</li>
+            <li class="template-item">API Design</li>
+            <li class="template-item">Version Control</li>
+            <li class="template-item">Web Development</li>
+          </ul>
+        </div>
+        <div id="dot-grid-template"><img src="Images SVG/image_dots.svg" alt=""></div>
+      </div>     
     </section>
 
     <section class="contact">
       <div class="headline">
-        <h2 class="title">Get started</h2>
-        <p class="myname-intro">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
+        <h2 class="title get">Get started</h2>
+        <p class="myname-intro started">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
       </div>
-
       <form action="https://formspree.io/f/xgebneoy" method="post">
+          <div class="name-email"> 
+            <div class="form-row">
+              <label for="name">Your name</label>
+              <input type="text" id="name" name="name" maxlength="30" placeholder="Write Your name..." required>
+            </div>
+            <div class="form-row">
+              <label for="email">Your email</label>
+              <input type="email" id="email" name="email" placeholder="Write Your email..." required>
+            </div>
+          </div>  
         <div class="form-row">
-           <label for="name">Your name</label>
-           <input type="text" id="name" name="name" maxlength="30" placeholder="Write Your name..." required>
+          <label id="label-textarea" for="contact-descrip">Your message</label>
+          <textarea id="contact-descrip" name="contact-descrip" maxlength="500" cols="30" rows="10" placeholder="Enter your message here..."></textarea>
```

Comment:
- [ ] Kindly check and fix the distance/margin between this section and the footer section. The form button seems to be pushing down the footer part. You need to follow the margins and positioning of the actual design 

- [ ] You also need to position the background image of this section very well since it is too close to the textarea. 
The goal of this project is to match the original figma design as much as possible. Kindly check and make these changes in order to achieve this purpose.

### Your design 

![Screenshot from 2023-04-08 00-55-13](https://user-images.githubusercontent.com/65233860/230697193-6d2657d1-7fb4-4cb1-b6af-1c1cba8021d0.png)



### Actual design 


![Screenshot from 2023-04-08 01-22-18](https://user-images.githubusercontent.com/65233860/230697215-16559de4-a951-4334-9d81-94754c36a73c.png)